RemoveWhiteSpace.java

```diff
@@ -0,0 +1,30 @@
+package com.company;
+
+import java.util.Scanner;
+
+import static com.company.Utils.println;
+import static com.company.Utils.print;
+
+public class RemoveWhiteSpace {
+
+    /*
+        removeWhiteSpace() method to remove the white space from an string
+        Input: character array containing all the elements of an string
+        Output: print the string without any whiteSpace
+         */
+    static void removeWhiteSpace(char[] inputStringArray) {
+        for (char stringChar : inputStringArray)
+            if (stringChar != ' ')
+                print(stringChar);
```

Comment:
here you print the same string with ignoring white space. please convert this string in a new one and return back to main method.